extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -59,9 +86,38 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
   }
 
   /** @override */
-  buildComponent() {
-    this.rootEl_ = buildSliderTemplate(this.element);
-    this.attachPrompt_(this.rootEl_);
-    return this.rootEl_;
+  layoutCallback() {
+    this.initializeListeners_();
+  }
```

Comment:
You don't need these 3 lines, since the `layoutCallback` from the abstract class already calls `initializeListeners_()`